### ISSUE TITLE
Fresh Docker install fails- 500 error 

### ISSUE THREAD

**Issue by r0nd0n** [AzuraCast/AzuraCast#2895]
**Using Docker installation method**
 
Yes
**AzuraCast version**

Cannot access
**Host Operating System**

Ubuntu 18.04- Fresh install from VPS
**Describe the bug**

After fresh Azuracast install on a fresh install of Ubuntu 18.04, and after following the 500 error issue 
#2855, I receive the following error after creating a username/pass:

![99(149)](https://user-images.githubusercontent.com/17818199/83251454-38867880-a16f-11ea-943d-dbe19da8dd01.png)

**To Reproduce**

1. Install Ubuntu 18.04, and update packages
2. Follow instructions from here: https://www.azuracast.com/install/docker.html
3. Apply fix for certificates found here: https://github.com/AzuraCast/AzuraCast/issues/2855
4. Create email/password for initial setup
5. Get the error in pic related.

**Expected behavior**

I've been using Azuracast for 2 years without any problems, and this problem has had my station down for 2 weeks now. I would expect it to get to the station dashboard once I created a superuser account.
**Relevant Logs**

I have went through the entire `docker-compose logs -f` and have found nothing that seems relevant, but can paste logs if needed.
**Screenshots**

**Device(s):**
 - Device:  Desktop
 - OS:  Windows 7
 - Browser:  Firefox
 - Version:  75.0

**Additional context**

I believe this to be a separate issue than #2855 but if I'm wrong I will close this thread and put this info there. Thank you.

**Comment by r0nd0n** [comment]
One additional tidbit of info:
During the install, this happened, but seemed to complete eventually:
```
2020/05/29 09:34:57 Waiting for: tcp://mariadb:3306
2020/05/29 09:34:57 Waiting for: tcp://influxdb:8086
2020/05/29 09:34:57 Waiting for: tcp://redis:6379
2020/05/29 09:34:57 Connected to tcp://influxdb:8086
2020/05/29 09:34:57 Problem with dial: dial tcp 172.18.0.4:3306: connect: connection refused. Sleeping 1s
2020/05/29 09:34:57 Connected to tcp://redis:6379
2020/05/29 09:34:58 Problem with dial: dial tcp 172.18.0.4:3306: connect: connection refused. Sleeping 1s
(above line repeats many times)
2020/05/29 09:35:37 Timeout after 40s waiting on dependencies to become available: [tcp://mariadb:3306 tcp://influxdb:8086 tcp://redis:6379]
```

**Comment by AaronUnmuted** [comment]
I also experienced this issue recently whilst attempting to install. To resolve, I actually followed the Docker update instructions as per here: https://www.azuracast.com/administration/system/updating.html#docker-installations

**Comment by r0nd0n** [comment]
> I also experienced this issue recently whilst attempting to install. To resolve, I actually followed the Docker update instructions as per here: https://www.azuracast.com/administration/system/updating.html#docker-installations

This actually solved my problem. I have installed/reinstalled and even used the ansible install method among a bunch of other things, and this has gotten me further than anything else. 

Thank you!
